model_package_core/tests/test_dataset.py

```diff
@@ -0,0 +1,28 @@
+from model_package_core.constants import (TRAIN_IMAGE_DATA_PATH,
+                                          TRAIN_LABEL_DATA_PATH)
+from model_package_core.pytorch_model.dataset import ProcessedDataset
+
+
+def test_processed_dataset_class():
```

Comment:
These test names aren't providing much useful context. It's good practice to name tests about its GIVEN/WHEN/THENs. Imagine this class becomes more sophisticated over time, and several unit tests are needed. What would you call the follow-up tests?
```suggestion
def test_dataset_normalizes_training_data_samples():
```

Reply:
Done.